src/main/java/onboarding/Problem5.java

```diff
@@ -1,11 +1,28 @@
 package onboarding;
 
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
 
 public class Problem5 {
     public static List<Integer> solution(int money) {
-        List<Integer> answer = Collections.emptyList();
+        List<Integer> answer = new ArrayList<>();
+        List<Integer> moneyUnit = new ArrayList<>() {{
+            add(50000);
+            add(10000);
+            add(5000);
+            add(1000);
+            add(500);
+            add(100);
+            add(50);
+            add(10);
+            add(1);
+        }};
```

Comment:
변하지 않는 값이라면 List.of를 활용하고 final 키워드를 붙여주면 더 좋을 것 같아요
```suggestion
        final List<Integer> moneyUnit = List.of(50000, 10000, ..., 1);
```